package.json

```diff
@@ -26,74 +26,75 @@
-    "@aws-cdk/aws-servicediscovery": "1.70.0",
-    "@aws-cdk/aws-sns": "1.70.0",
-    "@aws-cdk/aws-sns-subscriptions": "1.70.0",
-    "@aws-cdk/aws-sqs": "1.70.0",
-    "@aws-cdk/aws-ssm": "1.70.0",
-    "@aws-cdk/aws-stepfunctions": "1.70.0",
-    "@aws-cdk/cloud-assembly-schema": "1.70.0",
-    "@aws-cdk/core": "1.70.0",
-    "@aws-cdk/custom-resources": "1.70.0",
-    "@aws-cdk/cx-api": "1.70.0",
-    "@aws-cdk/region-info": "1.70.0",
-    "@types/jest": "^26.0.10",
-    "@types/node": "^14.6.1",
+    "@aws-cdk/assets": "1.72.0",
+    "@aws-cdk/aws-apigateway": "1.72.0",
+    "@aws-cdk/aws-applicationautoscaling": "1.72.0",
+    "@aws-cdk/aws-autoscaling": "1.72.0",
+    "@aws-cdk/aws-autoscaling-common": "1.72.0",
+    "@aws-cdk/aws-autoscaling-hooktargets": "1.72.0",
+    "@aws-cdk/aws-batch": "1.72.0",
+    "@aws-cdk/aws-certificatemanager": "1.72.0",
+    "@aws-cdk/aws-cloudformation": "1.72.0",
+    "@aws-cdk/aws-cloudfront": "1.72.0",
+    "@aws-cdk/aws-cloudwatch": "1.72.0",
+    "@aws-cdk/aws-cloudwatch-actions": "1.72.0",
+    "@aws-cdk/aws-codebuild": "1.72.0",
+    "@aws-cdk/aws-codecommit": "1.72.0",
+    "@aws-cdk/aws-codeguruprofiler": "1.72.0",
+    "@aws-cdk/aws-codepipeline": "1.72.0",
+    "@aws-cdk/aws-cognito": "1.72.0",
+    "@aws-cdk/aws-docdb": "1.72.0",
+    "@aws-cdk/aws-dynamodb": "1.72.0",
+    "@aws-cdk/aws-ec2": "1.72.0",
+    "@aws-cdk/aws-ecr": "1.72.0",
+    "@aws-cdk/aws-ecr-assets": "1.72.0",
+    "@aws-cdk/aws-ecs": "1.72.0",
+    "@aws-cdk/aws-ecs-patterns": "1.72.0",
+    "@aws-cdk/aws-efs": "1.72.0",
+    "@aws-cdk/aws-elasticloadbalancing": "1.72.0",
+    "@aws-cdk/aws-elasticloadbalancingv2": "1.72.0",
+    "@aws-cdk/aws-events": "1.72.0",
+    "@aws-cdk/aws-events-targets": "1.72.0",
+    "@aws-cdk/aws-iam": "1.72.0",
+    "@aws-cdk/aws-kinesis": "1.72.0",
+    "@aws-cdk/aws-kinesisfirehose": "1.72.0",
+    "@aws-cdk/aws-kms": "1.72.0",
+    "@aws-cdk/aws-lambda": "1.72.0",
+    "@aws-cdk/aws-logs": "1.72.0",
+    "@aws-cdk/aws-route53": "1.72.0",
+    "@aws-cdk/aws-route53-targets": "1.72.0",
+    "@aws-cdk/aws-s3": "1.72.0",
+    "@aws-cdk/aws-s3-assets": "1.72.0",
+    "@aws-cdk/aws-sam": "1.72.0",
+    "@aws-cdk/aws-secretsmanager": "1.72.0",
+    "@aws-cdk/aws-servicediscovery": "1.72.0",
+    "@aws-cdk/aws-sns": "1.72.0",
+    "@aws-cdk/aws-sns-subscriptions": "1.72.0",
+    "@aws-cdk/aws-sqs": "1.72.0",
+    "@aws-cdk/aws-ssm": "1.72.0",
+    "@aws-cdk/aws-stepfunctions": "1.72.0",
+    "@aws-cdk/cloud-assembly-schema": "1.72.0",
+    "@aws-cdk/core": "1.72.0",
+    "@aws-cdk/custom-resources": "1.72.0",
+    "@aws-cdk/cx-api": "1.72.0",
+    "@aws-cdk/region-info": "1.72.0",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^14.14.6",
     "@types/nodeunit": "^0.0.31",
-    "aws-cdk": "1.70.0",
-    "aws-sdk": "^2.741.0",
-    "constructs": "^3.0.4",
-    "conventional-changelog-cli": "^2.1.0",
+    "aws-cdk": "1.72.0",
+    "aws-sdk": "^2.787.0",
+    "constructs": "^3.2.15",
```

Comment:
CDK goes to a lot of effort to make sure they are pinning the version of `constructs` in all of their packages. I think we should try to always match their version and keep our linting rule the same as theirs. It looks like we missed this rule update in `pkglint` from 11 days ago: https://github.com/aws/aws-cdk/blob/master/tools/pkglint/lib/rules.ts#L1326

Our rule that needs updating is here: https://github.com/aws/aws-rfdk/blob/mainline/tools/pkglint/lib/rules.ts#L928

Reply:
Ok, I have changed `constructs` to version that used in CDK 3.2.0 instead of last version 3.2.15